lib/targets/cuda/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # generate an object library for all target specific files 
-add_library(quda_cuda_target OBJECT malloc.cpp tune.cpp)
+add_library(quda_cuda_target OBJECT malloc.cpp tune.cpp blas_cublas.cu)
```

Comment:
I admit this is something we need to look into but I had hoped not to have .cu files here. We might need to work out how to handle that, in particular making sure all flags are passed properly.

Reply:
@mathiaswagner this file at present is a .cu file since it uses a mini kernel to set some pointers.  If this is avoided, then this file can be cpp.

Reply:
I've addressed this: I got rid of the kernel, and now set the pointers on the host, and moved the file to cpp now.